src/component/admin/apiToken/ApiTokenForm/SelectProjectInput.tsx

```diff
@@ -0,0 +1,150 @@
+import React, { Fragment, FC, useMemo, useState } from 'react';
+import {
+    Link,
+    Checkbox,
+    FormControlLabel,
+    TextField,
+    Box,
+    Paper,
+} from '@material-ui/core';
+import { Autocomplete } from '@material-ui/lab';
+import CheckBoxOutlineBlankIcon from '@material-ui/icons/CheckBoxOutlineBlank';
+import CheckBoxIcon from '@material-ui/icons/CheckBox';
+import { IAutocompleteBoxOption } from 'component/common/AutocompleteBox/AutocompleteBox';
+import { useStyles } from './ApiTokenForm.styles';
+
+const ALL_PROJECTS = '*';
+
+const icon = <CheckBoxOutlineBlankIcon fontSize="small" />;
+const checkedIcon = <CheckBoxIcon fontSize="small" />;
```

Comment:
Should these be moved inside the component? I don't think it matters here, but would keep them in the tree.